sdk/ai/azure-ai-projects/samples/agents/async_samples/sample_agents_functions_async.py

```diff
@@ -37,6 +43,16 @@ async def main() -> None:
             # Initialize assistant functions
             functions = AsyncFunctionTool(functions=user_async_functions)
 
+            # fetch_weather function has a parameter named "unit" that can be either "Celsius" or "Fahrenheit"
+            # You can specify this constraint in the description of unit parameter to be picked up by OpenAI.
```

Comment:
Same comment about "OpenAI"

Reply:
Also, start comment line with capital letter

Reply:
use the word agents instead.